frontend/src/lib/user.ts

```diff
@@ -59,7 +59,7 @@ export function getHomePath(user: LexAuthUser | null): string {
   return isAdmin(user) ? '/admin' : '/';
 }
 
-export async function login(userId: string, password: string): Promise<LoginResult> {
+export async function login(userId: string, password: string, passwordStrength?: 0|1|2|3|4): Promise<LoginResult> {
```

Comment:
let's just remove `passwordStrength` here and call `zxcvbn` inside, that should simplify calling code